lib/doorkeeper/orm/mongoid7/stale_records_cleaner.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Doorkeeper
+  module Orm
+    module ActiveRecord
```

Comment:
`Mongoid7`

Reply:
@nbulaj I forgot to comment on the items you're commenting on.

At the time I wrote this, master in https://github.com/doorkeeper-gem/doorkeeper did not support an ORM specific StaleRecordsCleaner, so I wrote it this way to allow tests to pass.

I now see that a change was made to the doorkeeper gem to allow ORM versions of this class, and I will fix.

Thanks.